src/hooks/use-freshdesk-faqs.js

```diff
@@ -30,27 +29,33 @@ export const useFreshdeskFaqs = category => {
           `${FRESHDESK_API_ROOT}/solutions/categories/${categoryId}/folders`,
           requestOptions
         )
-        .then(response => {
+        .then((response) => {
+          console.log(response.data)
           const folders = response.data.sort((f, g) => f.name > g.name);
-          const articlePromises = folders.map(folder =>
+          folders.map((folder) =>
             axios
               .get(
                 `${FRESHDESK_API_ROOT}/solutions/folders/${folder.id}/articles`,
                 requestOptions
               )
-              .then(response => {
+              .then((response) => {
                 // A Freshdesk article's status is either 1 or 2, for draft or published, respectively
                 // We'll filter here for only the published articles.
                 folder.articles = response.data.filter(
-                  article => article.status === 2
+                  (article) => article.status === 2
                 );
                 return folder;
               })
+              .then((result) => {
```

Comment:
what's going on here? why are we passing in result but not using it?

Reply:
Left over from testing other strategies, removed now.